pom.xml

```diff
@@ -7,14 +7,14 @@
     <parent>
         <groupId>org.scijava</groupId>
         <artifactId>pom-scijava</artifactId>
-        <version>19.2.0</version>
+        <version>20.0.0</version>
         <relativePath />
     </parent>
 
     <groupId>sc.fiji</groupId>
     <!-- Name needs an underscore to be recognized as a plugin -->
     <artifactId>pQCT_</artifactId>
-    <version>2.0.1-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
 
```

Comment:
Great! Thanks for changing these. I almost missed these changes (and the deleted class), so they could have been separate commits.